src/hooks/usePublish.ts

```diff
@@ -7,23 +7,27 @@ export interface IUsePublishResponse {
   publish: () => void
 }
 
-export interface IUsePublishParams {
-  token: string
+export interface IUsePublishParams<
+  TokenType extends string | symbol = string | symbol,
```

Comment:
Thanks for the contribution!
Why not just declare ```TokenType extends string | symbol```?

Reply:
Good point, I somehow previously thought that it's needed. Removed & tested it a little bit and looks good!